modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/components/OrderableTable.tsx

```diff
@@ -265,24 +266,32 @@ const Row = ({
 						}
 					>
 						<ClayDropDown.ItemList>
-							{actions.map(({icon, label, onClick}) => (
-								<ClayDropDown.Item
-									key={label}
-									onClick={() =>
-										onClick({
-											item,
-										})
+							{actions.map(
+								({icon, isVisible, label, onClick}) => {
+									if (isVisible && !isVisible({item})) {
+										return;
```

Comment:
What would happen if all actions become hidden for an item? will a "no-op" ellipsis be rendered right?

Initially, I didn't believe we'll face this case in practise as we'll have `Delete` action at the very least, but considering permissions, we might have a case where `actions` only contains hidden items.

I have no strong opinion on this, in the end it's a product call. Technically, it's as easy as pre-filter the actions in line 252. No blocker though

Reply:
Yeah, as it's a product call, we can leave it for a followup.